README.md

```diff
@@ -1,34 +1,98 @@
+### Basic Ruby install
+In this project docker and docker-compose are managed through the ruby script found at `_docker/control.rb`. In order to run this you will require ruby 2.1.2. The following instructions install ruby via rbenv. You can use other methods, but your mileage may vary.
+#### Pre-requisits
+OSX:
+```bash
+brew install openssl libyaml libffi
+```
+
+Ubuntu/Debian/Mint:
+```bash
+apt-get install autoconf bison build-essential libssl-dev libyaml-dev libreadline6-dev zlib1g-dev libncurses5-dev libffi-dev libgdbm3 libgdbm-dev
+```
+
+Centos/Fedora:
+```bash
+yum install -y gcc openssl-devel libyaml-devel libffi-devel readline-devel zlib-devel gdbm-devel ncurses-devel
+```
+#### Installation of rbenv
+```bash
+git clone https://github.com/sstephenson/rbenv.git ~/.rbenv
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc #(or .bash_profile on a OSX)
+echo 'eval "$(rbenv init -)"' >> ~/.bashrc #(or .bash_profile on a OSX)
+source ~/.bashrc #(or .bash_profile on a OSX)
+```
+#### Installation of rbenv-build
+```bash
+git clone https://github.com/sstephenson/ruby-build.git ~/.rbenv/plugins/ruby-build
+```
+#### Installation of Ruby
+```bash
+rbenv install 2.1.2
+rbenv rehash
+```
+#### Setup ruby
+```bash
+gem install bundler
+```
+### Git checkout
 Fork the project, then clone your fork and add the upstream repository (Please ensure you have current version of git installed).
 
          git clone git@github.com:YOUR_USER_NAME/developers.redhat.com.git
          cd developers.redhat.com
          git remote add -f upstream git@github.com:redhat-developer/developers.redhat.com.git
 
-### Docker setup (Optional, but preferred)
-This section describes how to build the site using local instances of Searchisko (running in Docker containers).
-This section is optional, but highly recommended as this gives you a production-like environment locally.
-Skip to the [Site Build Setup](#site_build_setup) section if you don't want to use Docker.
+### Set up
+Download the Searchisko data dump from [here](https://github.com/redhat-developer/dcp-dumps/raw/master/searchisko.sql.zip) and copy to `_docker/searchisko/overlay/searchisko.sql.zip`
+
+Add the host `docker` to your `/etc/hosts` file. If you are building on Linux, set the IP address to `127.0.0.1`. If you are on a Mac and thus using Boot2Docker, you will need to set the IP address to that of your Boot2Docker image. You can discover this IP address by running `boot2docker ip`
+
+Run `bundle install` from within the `_docker` directory to download the necessary ruby gems.
+
+###Boot2Docker setup (OSX only)
+If you are running on Mac you will need to follow the steps below in _Edit your boot2docker DNS setup_ to setup configuration for the Red Hat DNS and VPN.
+
+If you are working on OSX or widows via boot to docker, then set the environment variable DOCKER_SSL_VERIFY to false. E.g `export DOCKER_SSL_VERIFY=false`. See here as to why https://github.com/swipely/docker-api/issues/202
```

Comment:
I've removed this altogether as the instructions don't support a windows install currently.